lms/templates/user_dropdown.html

```diff
@@ -2,6 +2,11 @@
 <%page expression_filter="h"/>
 <%namespace name='static' file='static_content.html'/>
 
+## This template should not use the target student's details when masquerading, see TNL-4895
+<%
+original_user = real_user or user
+%>
+
```

Comment:
Indeed there are. However, mako "helpfully" defines variables that don't exist as a special variable, `UNDEFINED`, instead of `None` as a sane person would expect, so I check for that now.

http://docs.makotemplates.org/en/latest/runtime.html#context-variables